introduction.md

```diff
@@ -1,3 +1,7 @@
 Hi there, I am Suhas learning Github from Code School ! 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?